tests/deletion_pool_test.go

```diff
@@ -1,7 +1,11 @@
 package tests
 
 import (
+	"sync"
 	"testing"
+
+	"github.com/ProtonMail/gluon"
+	events2 "github.com/ProtonMail/gluon/events"
```

Comment:
why call this events2? I don't see a clash anywhere.

Reply:
IDE auto-completion. Fixed.